code/go/0chain.net/blobbercore/handler/helper.go

```diff
@@ -13,8 +13,8 @@ func registerGRPCServices(r *mux.Router, server *grpc.Server) {
 	blobberService := newGRPCBlobberService()
 	grpcGatewayHandler := runtime.NewServeMux()
 
-	blobbergrpc.RegisterBlobberServer(server, blobberService)
-	_ = blobbergrpc.RegisterBlobberHandlerServer(context.Background(), grpcGatewayHandler, blobberService)
+	blobbergrpc.RegisterBlobberServiceServer(server, blobberService)
```

Comment:
naming issue too. please rename it if it is possible.

Reply:
Please see the above comment https://github.com/0chain/blobber/pull/252#discussion_r675301129